packages/iov-lisk/README.md

```diff
@@ -4,14 +4,61 @@
 
 ## Getting started
 
-This is how you use `liskCodec` and `LiskKeyringEntry` to generate send transactions
-for Lisk manually, i.e. without the help of @iov/core.
+The primary way to use @iov/lisk is together with @iov/core. Alternatively,
+you can use @iov/lisk to create offline transactions which can be posted manually.
 
-The following example is made for use in @iov/cli and you may need to include some
+All examples are made for use in @iov/cli and you may need to include some
 missing symbols if used in a different environment.
 
+### Using with @iov/core
+
+You can use @iov/lisk as an extension of @iov/core to interact with the
+Lisk blockchain as follows.
+
+```ts
```

Comment:
very nice that it works this way.
ideally also with other keyring entry, but cool to see the standard interfaces holding up.